matchms/filtering/metadata_processing/derive_ionmode.py

```diff
@@ -29,20 +29,20 @@ def derive_ionmode(spectrum_in: SpectrumType) -> SpectrumType:
 
     spectrum = spectrum_in.clone()
 
-    # Load lists of known adducts
-    known_adducts = load_known_adducts()
-
-    adduct = spectrum.get("adduct", None)
-    # Harmonize adduct string
-    if adduct:
-        adduct = _clean_adduct(adduct)
-
     ionmode = spectrum.get("ionmode")
     if ionmode:
         assert ionmode == ionmode.lower(), ("Ionmode field not harmonized.",
                                             "Apply 'make_ionmode_lowercase' filter first.")
```

Comment:
```suggestion
        assert ionmode == ionmode.lower(), "Ionmode field not harmonized."
```

Reply:
`make_ionmode_lowercase` no longer exists since this is now part of the `Metadata` class.

Reply:
I guess we can remove this assert statement altogether in this case.